docs/src/Fields/intro.md

```diff
@@ -24,8 +24,8 @@ various kinds of fields:
 - finite fields
 
 General textbooks offering details on theory and algorithms include:
-- [Cohen1](@cite)
-- [Cohen2](@cite)
+- [Coh93](@cite)
+- [Coh00](@cite)
 - [LN97](@cite)
-- [Marcus](@cite)
```

Comment:
OK, I just realized these changes first also need to be made in Hecke.jl (CC @thofma). I guess until then this PR needs to be on hold anyway, but that's fine, it's not urgent :-)

Reply:
Concerning the labels, see my remark in #637

Reply:
Ah sorry, wasn't aware. Thank you!